templates/user_profile_page.hbs

```diff
@@ -241,119 +241,162 @@
     {{/is}}
 
     {{#isnt current_filter.identifier 'activities'}}
-      <div class="container">
-        <section class="profile-section">
-
-          {{#if contributions}}
-
-            <header class="profile-section-header">
-              <h2 class="profile-section-title">{{t current_filter.identifier}}</h2>
-              {{#if sorters}}
-                <span class="profile-section-description">{{sorter_description}}</span>
-                <span class="profile-section-sorter dropdown">
-                  <button class="dropdown-toggle" aria-haspopup="true">
-                    {{current_sorter.label}}
-                    <svg xmlns="http://www.w3.org/2000/svg" width="12" height="12" focusable="false" viewBox="0 0 12 12" class="dropdown-chevron-icon">
-                      <path fill="none" stroke="currentColor" stroke-linecap="round" d="M3 4.5l2.6 2.6c.2.2.5.2.7 0L9 4.5"/>
-                    </svg>
-                  </button>
-                  <span class="dropdown-menu" role="menu">
-                    {{#each sorters}}
-                      <a href="{{url}}" aria-selected="{{selected}}" role="menuitem">
+      {{#is current_filter.identifier 'badges'}}
+        <div class="container">
+          <section class="profile-section">
+
+            {{#if user.badges}}
+              <header class="profile-section-header">
+                <h2 class="profile-section-title">{{t 'badges'}}</h2>
+                <span class="profile-section-description">{{t 'badges_description' name=user.name}}</span>
+              </header>
+
+              <div class="profile-badges-items">
+                {{#each user.badges}}
+                  <div class="profile-badges-item">
```

Comment:
Please use a semantic element (li or maybe section) for a11y. Also, an `aria-labelledby` attribute. See the Community list pages for example markup.

Reply:
Addressed in https://github.com/zendesk/copenhagen_theme/pull/229/commits/c77a68beeff100b7a3b355653a4e905cdceff812.

Reply:
Should I also use [aria-describedby](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-describedby_attribute) attribute pointing to the description of the badge?

Reply:
Yes, that sounds good to me 🙏

Reply:
Addressed in https://github.com/zendesk/copenhagen_theme/pull/229/commits/787c7fa863ff5806da2a4d1fd4fc41b0bea34c71.